drivers/emul/Kconfig

```diff
@@ -0,0 +1,49 @@
+# Emulator configuration options
+
+# Copyright 2020 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# Emulator options
+#
+menuconfig EMUL
+	bool "Emulation drivers"
+	select HAS_DTS_EMUL
+	help
+	  Enable Emulation Driver Configuration
+	  These drivers are used to emulate hardware devices, typically in
+	  native_posix, to support testing of various subsystems. For example,
```

Comment:
> typically in native_posix

Well.. only if that is your preferred debugging platform.
I mean, this works with any target with enough RAM. So I would not mention native_posix explicitly here, as you are not using any of its goodies or suffering any of its limitations.

Reply:
OK done